### PR TITLE
Removes PDA Chatrooms

### PR DESCRIPTION
**Pull request by razekpraxis**
Removes @tastyfish's PDA chatrooms.

While a nice feature, it's mainly only used as a tool for greytiders to
grief the telecomms server by adding everybody to a chatroom. THis
causes the comms to duplicate and forces command to shut down the PDA
server until it clears up.

It's also rarely ever actually used by anybody in a legitimate sense.

Debate down below if you want, a fair few people are behind me with
this.

### CODE REVIEW

File: code/game/machinery/computer/message.dm [hunk 1]
```diff
@@ -65,7 +65,7 @@
 			MK.loc = src.loc
 			playsound(loc, "sound/goonstation/machines/printer_dotmatrix.ogg", 50, 1)
 			// Will help make emagging the console not so easy to get away with.
-			MK.info += "<br><br><font color='red'>�%@%(*$%&(�&?*(%&�/{}</font>"
+			MK.info += "<br><br><font color='red'>�%@%(*$%&(�&?*(%&�/{}</font>"
```
Comment (Crazylemon64) [inline code/game/machinery/computer/message.dm]: Please use an editor that won't mangle unicode

**Comment by Crazylemon64** [comment]
If your issue is that people spamming "invite" can crash tcomms, perhaps what might be better is a rate limit.

File: code/game/machinery/computer/message.dm (hunk 1 above)
Comment (razekpraxis) [inline code/game/machinery/computer/message.dm]: mate i used Atom

**Comment by Crazylemon64** [comment]
You've got a leading merge commit, which is bad form for PR submission - to make a branch that is clean, for SURE:
(Make sure "upstream" is set up as a remote)
```
git fetch upstream
git checkout master
git status (make sure you don't have any changes in the live directory, they will be lost. Stash or commit them.)
git reset --hard upstream/master (caution: this WILL kill any commits on your "master" that are not in the mainline repo - if you're making changes in branches as you should be, you'll be fine)
```
Making new branches after this point will give you clean ones without merge commits or anything else on them.

**Comment by razekpraxis** [comment]
Maybe but that goes beyond my skills as a coder.

**Comment by razekpraxis** [comment]
WHY IS THERE A CHANGELOG COMMIT

**Comment by razekpraxis** [comment]
HELP ME IT KEEPS GETTING WORSE

**Comment by Anticept** [comment]
Or how about an ability to turn off making NEW rooms and forcefully closing existing ones? It DOES have a use when it's needed!

**Comment by razekpraxis** [comment]
Never needed! Never used!

**Comment by Crazylemon64** [comment]
> Never needed! Never used!

**Comment by Fox-McCloud** [comment]
You're going to have to follow @Crazylemon64's instructions and get rid of those unwanted commits if you want this to go through...

**Comment by razekpraxis** [comment]
@Fox-McCloud fixed them Kyet helpded me bless his gentle soul

**Comment by Iamgoofball** [comment]
"i cant fix the problem so instead of asking someone who knows what they're doing I'm just gonna kneejerk revert"

stop being a pussy and make the maybe 5 line change to fix this you fucking nerd

**Comment by Lady-Luck** [comment]
The PDA Chatrooms could be useful if they weren't so spammy. It would be better to make it more difficult to invite people (having to type their name out manually, for example) and making the chat alert noise default to off.

**Comment by Norgad** [comment]
Oh thank god, yes remove these. They are effectively just private radio channels.

**Comment by Plotron** [comment]
It's a nifty feature, shame about the performance issues.

**Comment by tristan1333** [comment]
imwithgoofball. Don't remove it. Fix it. They aren't that much of a problem in game anyway.

**Comment by ExitGame** [comment]
To be honest, it does add a advantage to Security when they i.e. want to communicate with other people and the captain alike let's say during nukeops.

But to be honest, they are abused on a daily basis, and if you can't fix them, remove them temporarily.

**Comment by Spacemanspark** [comment]
"Temporary" generally doesn't work out here.
I personally enjoy this feature and would rather it be fixed.

**Comment by Fox-McCloud** [comment]
Feature freeezee

**Comment by razekpraxis** [comment]
okay but i'm opening this again the nanosecond it's lifted

**Pull request closed without merging**